addons/edi/models/ir_attachment.py

```diff
@@ -0,0 +1,28 @@
+from odoo import api, models
+from odoo.exceptions import ValidationError
+from odoo.tools.translate import _
+
+import fnmatch
+
+class IrAttachment(models.Model):
+
+    _inherit = 'ir.attachment'
```

Comment:
Single quotes are used a lot in this file. Could you change them to double quotes? Running black on the file would do this for you as well

Reply:
This is a forked repo, so we should probably leave global formatting changes to upstream?